### PR TITLE
Revert "Update getting-started.md"

### PR DESCRIPTION
**Pull request by jbottum**
Reverts kubeflow/website#2934

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

File: content/en/docs/components/multi-tenancy/getting-started.md [hunk 1]
```diff
@@ -173,11 +173,12 @@ The following resources are created as part of the profile creation:
...
   - A Kubernetes namespace that shares the same name with the corresponding
     profile.
-  - Kubernetes RBAC ([Role-based access control](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)) role binding for the namespace: *Admin*. This makes the
+  - Kubernetes RBAC ([Role-based access control](https://kubernetes.io/docs/reference/access-authn-authz/rbac/))
+    role binding role binding for the namespace: *Admin*. This makes the
     profile owner the namespace administrator, thus giving them access to the
     namespace using kubectl (via the Kubernetes API).
   - Istio namespace-scoped AuthorizationPolicy: *user-userid-email-com-clusterrole-edit*.
-    This allows the `user` to access data belonging to the namespace the AuthorizationPolicy was created in 
+    This allows the `user` to access data beloging to the namespace the AuthorizationPolicy was created in
```
Comment (james-jwu) [inline content/en/docs/components/multi-tenancy/getting-started.md:181]: Typo: 'beloging'

**Comment by kimwnasptd** [comment]
Aside from the typo LGTM

**Comment by jbottum** [comment]
@shannonbradshaw I need help on this.  I have not reverted and cant tell if James has issued the typo change.   what do I need to do ?

**Comment by shannonbradshaw** [comment]
/close

This was handled in #3079

**Pull request closed without merging**